src/templates/udmcfg.yaml.j2

```diff
@@ -0,0 +1,25 @@
+configuration:
+  keys:
+    udmProfileAHNPrivateKey: c53c22208b61860b06c62e5406a7b330c2b577aa5558981510d128247d38bd1d
+    udmProfileAHNPublicKey: 5a8d38864820197c3394b92613b20b91633cbd897119273bf8e4a6f4eec0a650
+    udmProfileBHNPrivateKey: F1AB1074477EBCC7F554EA1C5FC368B1616730155E0041AC447D6301975FECDA
+    udmProfileBHNPublicKey: 0472DA71976234CE833A6907425867B82E074D44EF907DFB4B3E21C1C2256EBCD15A7DED52FCBB097A4ED250E036C7B9C8C7004C4EEDC4F068CD7BF8D3F900E3B4
+  nrfUri: {{ nrf_url }}
+  sbi:
+    bindingIPv4: 0.0.0.0
+    port: {{ udm_sbi_port }}
+    registerIPv4: {{ udm_hostname }}
```

Comment:
I know there's a DNS somewhere, but doesn't it feel kinda strange that we pass the hostname to the param that says `IPv4`?

Reply:
Well yes, I agree it feels kinda odd. However is there any other thing we can do? They do the same. If you look at the [config files](https://github.com/omec-project/smf/blob/3fa04ea13dc4c8207a2d93b6fb4b34d562748f46/config/smfcfg.yaml#L41) there's a hardcoded service name there. What would you suggest?

Reply:
I used pod's IP address.

Reply:
Then I'd say we must agree as a team, we've been using this approach for all the charms, and it feels odd to use different approaches for different charms, if there are connectivity issues, won't be the best troubleshooting experience.

Reply:
Changed as discussed.